server/context.go

```diff
@@ -165,12 +166,12 @@ func (ctx *Context) initEngine(attrsStr, path string) (engine.Engine, error) {
 	return engine.NewRocksDB(attrs, path, ctx.CacheSize), nil
 }
 
-// parseAttributes parses a colon-separated list of strings,
+// parseAttributes parses a list of strings separated by sep,
 // filtering empty strings (i.e. ",," will yield no attributes.
 // Returns the list of strings as Attributes.
-func parseAttributes(attrsStr string) proto.Attributes {
+func parseAttributes(attrsStr string, sep string) proto.Attributes {
```

Comment:
Hmm, we can use `:` instead of `,` as separator of --attrs, but I thought that it's a bit uncommon convention. 

We cannot simply change the separator of --stores since the flag uses both `:` and `,` as its separator. The flag description says "specify a comma-separated list of stores, specified by a colon-separated list of device attributes ...".